crud/src/test/resources/ITCaseCrudResourceTest/config/config.properties

```diff
@@ -1,4 +1,4 @@
 # set default isolation strategy for tests to SEMAPHORE else breakpoints in IDE
 # when debugging unit tests will timeout with default strategy (THREAD).
 hystrix.command.default.execution.isolation.strategy=SEMAPHORE
-hystrix.command.default.execution.isolation.thread.timeoutInMilliseconds=60000
+hystrix.command.default.execution.isolation.thread.timeoutInMilliseconds=120000
```

Comment:
Increased to debug many different places without hystrix interrupting the thread.  Can be reverted if you see a big issue (which I would suggest to we make explicit note on the test that we have time constraint). Note that the tests were used to have unlimited time before hystrix 1.4.0